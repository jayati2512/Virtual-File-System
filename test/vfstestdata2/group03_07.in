createvfs demo 1024
mountvfs demo
mountvfs demo
makedir / home
makedir /home/d1 d2
makedir /home/d1/d2 d3
makedir /home/d1/d2/d3 d4
makedir /home/d1/d2/d3/d4 d5 
makedir /home/d1 d2    
deletedir /home/d1/d2
makedir /home/d1/d2 
addfile /home/d1/d2 file1.txt /usr/include/stab.h
updatefile /home/file2.txt /usr/include/stab.h
addfile /home/d1 file2.txt /usr/include/stab.h
addfile /home/d1/d2/d3 file3.txt /usr/include/stab.h
copyfile /home/file /home/d1/file1.txt  
unmountvfs demo
mountvfs demo
mountvfs demo 
listdir /home 1 /tmp/output_list_dir.txt
removefile /home/example.txt 
removefile /home/d1/d2/file1.txt
removefile /home/d1/d2/d3/file3.txt
movefile /home/d1/d2/d3/file3.txt /tmp/output.txt 
unmountvfs demo
unmountvfs demo 
