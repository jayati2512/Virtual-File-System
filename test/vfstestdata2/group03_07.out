createvfs_SUCCESS
mountvfs_SUCCESS
mountvfs_FAILURE VFS_ALREADY_MOUNTED
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
makedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
addfile_SUCCESS
updatefile_FAILURE INTERNAL_FILE_NOT_FOUND
addfile_SUCCESS
addfile_SUCCESS
copyfile_FAILURE CANNOT_FIND_SPECIFIED_SOURCEFILE
unmountvfs_SUCCESS
mountvfs_SUCCESS
mountvfs_FAILURE VFS_ALREADY_MOUNTED
listdir_SUCCESS
removefile_FAILURE CANNOT_FIND_SPECIFIED_PATH_OR_FILE
removefile_SUCCESS
removefile_SUCCESS
movefile_FAILURE CANNOT_FIND_SPECIFIED_SOURCEFILE
unmountvfs_SUCCESS
unmountvfs_FAILURE VFS_NOT_MOUNTED
